### ISSUE TITLE
Unable to open Job Details link from Workflow Visualizer while Job is running

### ISSUE THREAD

**Issue by unlikelyzero** [ansible/awx#2606]
##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME
 - UI

##### SUMMARY
Unable to open Job Details link from Workflow Visualizer while Job is running. Link works as soon as job finishes. No console errors or Network traffic.

##### ENVIRONMENT
* AWX version: 3.4.0
* Operating System: macOS
* Web Browser: Firefox and Chrome

##### STEPS TO REPRODUCE
1. Create a WFJT which contains WFJTs or JTs
2. Run WFJT.
3. While JT is running, click on 'DETAILS' link

##### EXPECTED RESULTS

JT Detail view is opened

##### ACTUAL RESULTS

Nothing

##### ADDITIONAL INFORMATION

![chatty](https://user-images.githubusercontent.com/12446869/48042469-9a2b6000-e14f-11e8-8ea9-b99e2a62069c.gif)

**Comment by unlikelyzero** [comment]
@ryanpetrello was this reproducible only when using concurrent_execution enabled?

**Comment by kdelee** [comment]
FWIW I'm still getting this bug today on branch up to date with devel

**Comment by dsesami** [comment]
Reopening for verification by @unlikelyzero

**Comment by elyezer** [comment]
I tried to reproduce on a fresh install and the details links are working as expected, here are the scenarios I tried on both Firefox and Chrome:

1) Workflow with a single job
2) Workflow with 3 job running in parallel
3) Workflow with 3 jobs running in parallel, then after all 3 being completed another job.
4) The same as 3 but having a parent workflow to call the above's workflow.

This is now verified.
